project/plugins.sbt

```diff
@@ -12,3 +13,10 @@ addSbtPlugin("com.github.sbt"    % "sbt-dynver"   % "5.0.1")
 // This project is its own plugin :)
 Compile / unmanagedSourceDirectories += baseDirectory.value.getParentFile / "src" / "main" / "scala"
 Compile / unmanagedResourceDirectories += baseDirectory.value.getParentFile / "src" / "main" / "resources"
+
+dependencyOverrides ++= Seq(
+  "org.eclipse.jgit"          % "org.eclipse.jgit" % "6.8.0.202311291450-r", // sbt-scalafix
+  "ch.qos.logback"            % "logback-core"     % "1.4.7", // sbt-sonatype
+  "com.google.guava"          % "guava"            % "23.0", // sbt-sonatype
+  "org.apache.httpcomponents" % "httpclient"       % "4.5.14" // sbt-sonatype
+)
```

Comment:
Is there no way to share this list instead of duplicating?

Reply:
We already share the list of sbt plugins.

If you find a way, I would love to!

Reply:
Added a commit...
![](https://media1.tenor.com/m/uWUfFiFuSX0AAAAC/inception-nickelback.gif)
(and scalafmt it)